Improve "Cannot find .abapgit.xml" situation

mbtools opened the pull request:
Skip check during creating of repo and increase file count.

Closes #4174

larshp reviewed (changes requested):
Comments below

larshp commented:
I dont think this should be a special case, it should be generic error handling from the create repo page.

1: User opens the repo page, enters data
2: Click create/clone

In my case the repository was created, but the UI stayed on the create repo page. If the repository is created, then the UI should show the repository, if not created then show error.

I have not looked into the details, but I guess there are more scenarios than the `.abapgit.xml` file issue

mbtools commented:
maybe this check should be a sep method that's called from the UI only

```abap
find_remote_dot_abapgit

    ELSEIF lines( mt_remote ) > c_new_repo_size.
      " Less files means it's a new repo (with just readme and license, for example) which is ok
      zcx_abapgit_exception=>raise( |Cannot find .abapgit.xml - Is this an abapGit repo?| ).
```

larshp commented:
hmm, yea, but still the repository is created when the exception occurs

I think,
a: move the error mentioned above, as it should just be a warning
b: there is something wrong with general error handling during creation and the UI flow, might be outside scope for this PR

mbtools commented:
- Removed exception during `new_online`
- Added warning to repo view
![image](https://user-images.githubusercontent.com/59966492/101423956-262c8a80-38c8-11eb-8add-2b57e1dd42b5.png)

larshp commented on src/zcl_abapgit_repo_content_list.clas.abap:
```diff
@@ -190,6 +194,27 @@ CLASS ZCL_ABAPGIT_REPO_CONTENT_LIST IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD check_for_dot_abapgit.
```
suggest renaming this method to "check_repo_size" ?
